app/helpers/index_helper.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+module IndexHelper
+  def header_options(id)
+    options = [
+      "Application number",
+      "Site address",
+      "Application type",
+      "Expiry date",
+      "Days left to expiry",
+      "Status"
+    ]
```

Comment:
Could be a constant?

Reply:
I'm modifying it though in the next few lines though, isn't it a bit weird to be modifying a constant?